doc/api/http.md

```diff
@@ -2173,7 +2173,8 @@ changes:
     description: The `options` parameter can be a WHATWG `URL` object.
 -->
 
-* `url` {string | URL}
+* `url` {string | URL} `URL` here is not the legacy
+  `urlObject`(`require('url').Url`), but an instance of the `URL` class.
```

Comment:
I think I know what is being expressed here. Are you referring to one of the following?

- https://developer.mozilla.org/en-US/docs/Web/API/URL
- https://nodejs.org/docs/latest-v14.x/api/url.html

If so, you may want to consider linking to them. For clarity, I think it would be great to link to the actual ECMAScript specification, but I don't know how things have been previously been handled in situations like these. It's an interesting situation to be in. I will have to think more about this, but can't guarantee I will have a feasible solution. Perhaps take inspiration from other places as well.

- https://github.com/nodejs/node/issues/33977

I'm also not familiar enough w/ the context of this change or anything else. It did seem to be that you ran into a similar problem as myself. Hope that was helpful and would be interested in the outcome. At the very least, linking to somewhere would be a good idea though. Where to?

Reply:
At the moment I've followed the convention and linked these back to the node docs themselves, just to hint the users about their usage specific to Node.js. Wdyt?